Userland/Utilities/try-override-graphics.cpp

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
This seems to be some testing code that got committed by accident?

Reply:
Not really, I wanted to put that as a test, but CI doesn't run in GUI mode, so I'm not sure what to do with this small utility.

Reply:
It seems like there's no good place for this test program to reside in, so I'll remove it and mark this resolved :)